docker-compose.yml

```diff
@@ -1,81 +1,59 @@
-version: '2.1'
-
-networks:
-  get5:
-    external: true
+version: "3.7"
 
 services:
-  caddy:
-    image: caddy:2-alpine
-    container_name: caddy-reverse-proxy
-    restart: unless-stopped
+  redis:
+    image: redis:6
+    container_name: redis
+    restart: always
     ports:
-      - "80:80"
-      - "443:443"
-    volumes: 
-      - ./Caddyfile:/etc/caddy/Caddyfile
-    networks:
-      - get5
-    environment:
-      - CADDY_URL=
-      - CADDY_API_ENDPOINT=api
-      - CADDY_REVERSE_PROXY_PORT=3301
-  
+      - 6379:6379
+
   get5db:
-    image: mariadb
+    image: mysql
```

Comment:
I would suggest staying with mariadb

Reply:
Ah yes, forgot this as well. Please stick with mariadb :) that's done by choice.